### PR TITLE
exercise 3.9 -3.11

### PR DESCRIPTION
**Pull request by idobalul**

### CODE REVIEW

File: apiRouter.js [hunk 1]
```diff
@@ -1,49 +1,51 @@
...
-    if (index || index === 0) {
-        peopleData.splice(index, 1)
-        res.send('Contact deleted')
-    } else {
-        res.status(404).send('ID not found or not exists')
-    }
-})
+  const id = Number(req.params.id);
+  const index = peopleData.findIndex((i) => i.id === id);
+  if (index || index === 0) {
```
Comment (ofirsalem1) [inline apiRouter.js:21]: what if the index is not exsist

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+const list = document.createElement('ul'); //for the output
+
+/****************
+ * API requests *
+ ****************/
+async function getInfo() {
+  try {
+    clearOutput();
+    const response = await axios.get(`${BASE_URL}/info`);
+    output.innerHTML = response.data;
```
Comment (ofirsalem1) [inline src/index.js:26]: recomended not to use innerHTML

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+    for (const person of response.data) {
+      list.append(createContactElement(person.id, person.name, person.number));
+    }
+    output.append(list);
+  } catch (error) {
+    output.innerText = error.response.data;
+  }
+}
+
+async function getPerson(id) {
```
Comment (ofirsalem1) [inline src/index.js:45]: need to check the input

File: apiRouter.js (hunk 1 above)
Comment (idobalul) [reply apiRouter.js:21]: תוקן

File: src/index.js (hunk 2 above)
Comment (idobalul) [reply src/index.js:26]: תוקן

File: src/index.js (hunk 3 above)
Comment (idobalul) [reply src/index.js:45]: תוקן

**Pull request closed without merging**